bindings/rust/generate.sh

```diff
@@ -0,0 +1,31 @@
+#/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# cd into the script directory so it can be executed from anywhere
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+mkdir -p s2n-tls-sys/lib
+
+# we copy the C sources into the `lib` directory so they get published in the
+# actual crate artifact.
+cp -r \
+  --update \
```

Comment:
--update is not available on BSD/OSX, consider rsync?

Reply:
ah good call. I'll just remove it as it was mostly an optimization.